.github/workflows/automatic-release.yml

```
@@ -3,8 +3,8 @@ on:
   workflow_call:
     inputs:
       NODE_VERSION:
-        description: Node version with which the release will be executed.
-        default: 18
+        description: Node version with which the release will be executed (deprecated).
+        default: 20
```

Since we deprecate this input and no longer use it, changing it is unnecessary, IMO.
```suggestion
        default: 18
```

---

As you suggested, I directly removed the input.